package.json

```diff
@@ -470,6 +495,16 @@
           "command": "metals.new-scala-file",
           "when": "metals:enabled",
           "group": "navigation@4"
+        },
+        {
+          "command": "metals.show-javap-verbose",
```

Comment:
We could probably have a submenu called `Analyse Scala file` or something along those lines? 

https://code.visualstudio.com/api/references/contribution-points#contributes.menus